session/manager.go

```diff
@@ -0,0 +1,27 @@
+package session
+
+type Manager struct {
+	Generator GeneratorInterface
+	sessions  []SessionId
+}
+
+func (manager *Manager) Add(sid SessionId) {
+	manager.sessions = append(manager.sessions, sid)
+}
+
+func (manager *Manager) Create() SessionId {
+	id := manager.Generator.Generate()
+	manager.Add(id)
+
+	return id
+}
+
+func (manager *Manager) Get(sid SessionId) (sessionId SessionId) {
```

Comment:
Impossible to distinguish if session was found/not-found.
Test would help You to prove usability.